CSharp/BatchExplorer/ViewModels/MainViewModel.cs

```diff
@@ -1014,6 +1014,29 @@ public CommandBase OpenRDP
         }
 
         /// <summary>
+        /// Get SSH settngs for the selected ComputeNode
+        /// </summary>
+        public CommandBase GetSSH
```

Comment:
Minor: Isn't the correct capitalization convention here `GetSsh`? I thought that acronyms were not all-caps unless they were only 2 characters long?